Gtk-3.0.gir

```diff
@@ -56656,7 +56656,7 @@ changed, the image will be updated automatically.</doc>
         <type name="IconSet"/>
       </property>
       <property name="icon-size" writable="1" transfer-ownership="none">
-        <type name="gint" c:type="gint"/>
+        <type name="IconSize" c:type="GtkIconSize"/>
```

Comment:
Same in `GtkStackSwitcher`:

```xml
      <property name="icon-size" version="3.20" writable="1" transfer-ownership="none">
        <doc xml:space="preserve">Use the "icon-size" property to change the size of the image displayed
when a #GtkStackSwitcher is displaying icons.</doc>
        <type name="gint" c:type="gint"/>
      </property>
```

That seem to be all though.

Reply:
Good catch, thanks!